gem/templates/springster/patterns/basics/surveys/surveys-modules/form.html

```diff
@@ -64,9 +64,9 @@
     </div>
   </fieldset>
   {% endfor %}
-  <!--Find a way to include {{surveytype}} variable to text -->
+
   {% trans "Submit" as text %}
-  <input type="submit" value="{% if is_intermediate_step %}{% trans 'Next Question' %}{% else %}{{ self.submit_text|default:text}} {{surveytype}}{% endif %}" class="call-to-action__button call-to-action__button--primary" />
+  <input type="submit" value="{% if is_intermediate_step %}{% trans 'Next Question' %}{% elif survey.display_survey_directly %}{{ self.submit_text|default:"Vote on poll" }}{% else %}{{ self.submit_text|default:text|add:" "|add:surveytype}} {% endif %}" class="call-to-action__button call-to-action__button--primary" />
```

Comment:
should `Vote on poll` not be in a trans tag as well?

something like
`{% trans "Vote on poll" as defaulttext %}`
`{{ self.submit_text|default:defaulttext }}`